src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
-}
+	public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.